lib/Email/Outlook/Message.pm

```diff
@@ -105,7 +105,6 @@ our $MAP_CODEPAGE = {
   20127 => 'US-ASCII',
   20866 => 'KOI8-R',
   28591 => 'ISO-8859-1',
-  28592 => 'ISO-8859-2',
```

Comment:
This doesn't seem related to your change, or a desirable thing to remove.

This entry was only added recently, so maybe a patch rebasing error?